docs/docs/notification-center/react-components.md

```diff
@@ -212,7 +212,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>tr</code> (Turkish)</li>
           <li><code>te</code> (Telugu)</li>
           <li><code>vi</code> (Vietnamese)</li>
-          
+          <li><code>zo</code> (Zulu)</li>
```

Comment:
Zulu ISO code is `zu`.

Reply:
I'll fix it

Reply:
And talking about "Notifications" , In Zulu they call it isaziso (Notice) @p-fernandez